awx/main/management/commands/revoke_tokens.py

```diff
@@ -0,0 +1,64 @@
+# Django
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+from django.utils import timezone
+
+
+# AWX
+from awx.main.models.oauth import OAuth2AccessToken
+from oauth2_provider.models import RefreshToken
+
+
+def revoke(token):
```

Comment:
We should be able to do without this revoke() method now that [this issue](https://github.com/jazzband/django-oauth-toolkit/issues/625) is fixed in our fork of DOT.